assets/css/style.css

```diff
@@ -0,0 +1,198 @@
+/* Reset Styling */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  width: 100%;
+  font-size: 20px;
+  font-family: lato, nyt-imperial, georgia, "times new roman", times, serif;
+  background-color: #fff;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+ul li {
+  text-decoration: none;
+  list-style: none;
+}
+
+.clearfix::after {
+  content: ".";
+  visibility: hidden;
+  display: block;
+  clear: both;
+}
+
+/* Navigation */
+
+nav {
+  width: 100%;
+}
+
+nav ul {
+  display: flex;
+  margin-top: 100px;
+  border: 1px solid #444;
+  height: 50px;
+  border-radius: 5px;
+  background: linear-gradient(#7c807c 50%, #5f635f 50%);
+  color: #fff;
+  overflow: hidden;
+}
+
+nav ul li {
+  padding: 10px 30px;
+  border-left: 1px solid #fff;
```

Comment:
Why are we using white border left here? :thinking: Also, we could add hover effects as in the [Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/).

![preffered apple nav hover](https://user-images.githubusercontent.com/36057474/62406543-8cebb680-b5a5-11e9-90c7-d795f787c77e.gif)